texttospeech/ssml_addresses/resources/example.ssml

```diff
@@ -0,0 +1,3 @@
+<speak>123 Street Ln, Small Town, IL 12345 USA
```

Comment:
It doesn't look like this file is used at all, so might be worth removing. 

Or remove the test files below and use the `example.txt` and `example.ssml` in your test files as these seem to more accurately test the tutorial for its intended purpose.

Reply:
Thanks for the suggestion. I went ahead and reconfigured the tests to just use the example files.